tests/components/azure_data_explorer/test_client.py

```diff
@@ -0,0 +1,87 @@
+"""Test the init functions for AEH."""
+import logging
+
+from homeassistant.components import azure_data_explorer
+from homeassistant.components.azure_data_explorer.client import AzureDataExplorerClient
+
+from .const import BASE_CONFIG_FULL, BASIC_OPTIONS
+
+from tests.common import MockConfigEntry
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def test_managedstreaming_ingestion(mock_managed_streaming):
+    """Test ManagedStreaming ingestion."""
+
+    entry = MockConfigEntry(
+        domain=azure_data_explorer.DOMAIN,
+        data=BASE_CONFIG_FULL,
+        title="cluster",
+        options=BASIC_OPTIONS,
+    )
+    client = AzureDataExplorerClient(
```

Comment:
Tests shouldn't use the internal workings of the integration, so add the entry to hass, and then interact with it by adding a event to the queue and advancing the time so that a send event is done (using the mock streaming client) and then check whether it did any sending!